modules/mobianRtdProvider.js

```diff
@@ -70,6 +71,18 @@ function getBidRequestData(bidReqConfig, callback, config) {
         deepSetValue(ortb2Site, 'ext.data.mobianGenres', genres);
         deepSetValue(ortb2Site, 'ext.data.apValues', apValues);
 
+        setKeyValue('mobianRisk', mobianRisk);
+        setKeyValue('mobianContentCategories', contentCategories);
+        setKeyValue('mobianSentiment', sentiment);
+        setKeyValue('mobianEmotions', emotions);
+        setKeyValue('mobianThemes', themes);
+        setKeyValue('mobianTones', tones);
+        setKeyValue('mobianGenres', genres);
+        setKeyValue('ap_a0', apValues.a0 || []);
+        setKeyValue('ap_a1', apValues.a1 || []);
+        setKeyValue('ap_p0', apValues.p0 || []);
+        setKeyValue('ap_p1', apValues.p1 || []);
```

Comment:
I don't believe the single value solution works for our clients to be able to properly target from GAM, which is what we need. Likewise, the key names should not be configurable but predefined, for the same purpose.

Reply:
Could you indicate how you would target from gam using your suggestion, and also what you mean with configurable keynames specifically? Can you provide examples?

Reply:
On configuration - the first parameter passed to `init` is configuration as set by the publisher. You could accept a `prefix` parameter that controls the key names and/or turns them off. for example

```
pbjs.setConfig({
    realTimeData: {
        dataProviders: [{
            name: 'mobianBrandSafety',
            params: {
                prefix: 'custom'
            }
        }]
    }
})
```

would give you `{name: 'mobianBrandSafety', params: { prefix: 'custom' }}` as the first argument to `init`; and you could use that to set e.g. `customSentiment` instead of `mobianSentiment`. Setting `null` could turn it off.

More parameters could be used to turn off / customize only some keys, but how to do that exactly depends on what makes sense for your use case. You could have one for each ( `sentiment: 'customSentiment', emotion: 'customEmotion'`, etc), or it may make more sense to separate them into clusters. Without understanding exactly what you're doing, I'd expect the `ap_` keys to be different from the rest for example.

Reply:
Updated code to enable configuration, and updated tests accordingly. Looking forward to your comments.